docs/connectors.md

```diff
@@ -0,0 +1,47 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Kafka and Amazon S3 connectors.
+
+The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.
+
+
+## Roadmap & Feedback
+
+Based on what the Conduit team hears from developers, a [roadmap]() has been built to facilitate the connectors that need to be built for Conduit. If you have a connector that you really want to see supported or lend your vote for a particular connector, check out that roadmap and upvote the associated issue!
```

Comment:
Nitpick: link for roadmap missing.

Reply:
fixed

Reply:
argh.

Reply:
I've approved, feel free to merge when you push this change too.

Reply:
fixed.